.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Prerequisites**
+
+* [ ] I've updated the `@thirdweb-dev/sdk` package to the latest version.
+* [ ] I've checked other issues and it's not duplicated, check the [issues list](https://github.com/thirdweb-dev/typescript-sdk/issues?q=is%3Aissue+is%3Aclosed).
+
+**Optional**
+
+* [ ] I am willing to contribute a PR that fixes the problem with some guidance.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Install '...'
+2. Initialize '....'
+3. Execute the function '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Package versions (please complete the following information):**
+ - `@thirdweb-dev/sdk`: [e.g. 2.2.6]
+ - `node`: [e.g. v12.16.3]
+ - Frontend of your choice if applicable (`react`, `next`, `angular`, `vue`, etc.): [e.g. v12.16.3]
```

Comment:
We really care about any build tools, etc here instead

Reply:
as in, in terms of "fronend of your choice"

Reply:
we want `sdk version` + `node version` + `any build tools, typescript vs not, etc`